jwst/stpipe/core.py

```diff
@@ -16,7 +16,7 @@
 class JwstStep(Step):
 
     spec = """
-    output_ext         = string(default='.fits')     # Default type of output
+    output_ext = string(default='.fits')  # Default output suffix, not output type
```

Comment:
Ok, so 'suffix' is an overloaded term to avoid here - yes, I now see that from other areas of the docs.   I was trying to make it clear that `output_ext` described the ascii characters of the filename to the right of the decimal part of the filename ( the 'extension' ) and not the underlying type.

Are 'filename extension` or just 'extension' unambigous, such as :

  output_ext = string(default='.fits')  # Default output extension

 I see that white_light_step also currently has:

 output_ext         = string(default='.ecsv')  # Default type of output

 . . . so I can also update that to :

#  Default output extension

Reply:
Because this applies to the type/format of the disk file that's created, as opposed to the type of data in the file, I think it would be good to explicitly use the term "file" somewhere in comment string. And I don't think we need to include the word "Default" in there, because this is a parameter that users can change and they can easily see what the default value is by looking at the rest of the definition. So how about "Output file type" for both instances of `output_ext` shown above. Or if we really want to work the word "extension" in there, to make the connection with the "_ext" portion of the param name, we could go with "Output file name extension".